dao/mongodb.rya/src/main/java/org/apache/rya/mongodb/MongoDBRyaDAO.java

```diff
@@ -93,7 +93,7 @@ public MongoDBRyaDAO(final MongoDBRdfConfiguration conf, final MongoClient mongo
     @Override
     public void setConf(final MongoDBRdfConfiguration conf) {
         this.conf = conf;
-        this.auths = conf.getAuthorizations();
+        auths = conf.getAuthorizations();
```

Comment:
it was removed due to a save action deeming it unnecessary.  I'll add it back in